docs/how_to_use_prefixed_service_in_custom_rule.md

```diff
@@ -0,0 +1,39 @@
+# How to Use Prefixed Service In Custom Rule
+
+Since `rector/rector` version 0.11, it is not possible to use service that previously not prefixed in previous version, for example, you have the following custom rector rule with `Symplify\PackageBuilder\Strings\StringFormatConverter` dependency:
+
+```php
+use Symplify\PackageBuilder\Strings\StringFormatConverter;
+
+final class UnderscoreToCamelCaseVariableNameRector extends AbstractRector
+{
+    public function __construct(StringFormatConverter $stringFormatter)
+    {
+        // ...
+    }
+}
+```
+
+For above example, the `Symplify\PackageBuilder\Strings\StringFormatConverter` is no longer exists, you can consume via require --dev it:
+
+```bash
+composer require --dev symplify/package-builder
+```
+
+After that, you need to register the `symplify/package-builder`'s src to service in `rector.php`:
+
+```php
+<?php
+// rector.php
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // ...
+
+	$services = $containerConfigurator->services();
+	$services->load('Symplify\\PackageBuilder\\', __DIR__ . '/vendor/symplify/package-builder/src');
+
+    // ...
+};
```

Comment:
Here config can be improved instead.
Even more generic and Symfony best practice is to register service manually.

That can be used in any PHP class case

Reply:
implemented https://github.com/rectorphp/rector-src/pull/33/commits/b571141c815ee70ab939fb512e33533dd5f028fe